default.nix

```diff
@@ -154,6 +162,7 @@
         hostName = "localhost";
         systems = config.simple-hydra.localBuilder.systems;
         maxJobs = config.simple-hydra.localBuilder.maxJobs;
+	supportedFeatures = config.simple-hydra.localBuilder.supportedFeatures;
```

Comment:
Indentation ;)

Reply:
Oh, weird!  That's what i get for pushing from a machine with no emacs config :P